appveyor.yml

```diff
@@ -29,7 +29,8 @@ environment:
       CI_TYPE: INTEGRATION
 
 install:
-  - ps: .\appveyor_install.ps1
+  - ps: .\build\appveyor_install.ps1
```

Comment:
Since the file was moved, we need to use this path as cache dependency [(line 8)](https://github.com/datastax/csharp-driver/pull/293/files#diff-180360612c6b8c4ed830919bbb4dd459R8).